feat: Enhance the overall user experience of the prime process and occupy new features

yikZero opened the pull request:
The introduction pictures and descriptions of the account activity limit increase and the history export function will be provided next week, and the i18n will be replaced at that time.

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* New Features
  * Added Prime features: Notifications and History Export with icons, descriptions and contextual actions.
* UI/UX
  * PrimeDashboard gets a top-left close control; banner now reflects active Prime status.
  * Bulk Revoke icon updated; Prime badge shown only to non‑Prime users.
* Bug Fixes
  * Improved modal stability by closing the triggering action first and adding a short delay.
* Translations
  * Added keys for history export, multi‑account notifications, on‑chain notifications description, Prime unlock description, and transaction export description.
* Refactor
  * Simplified modal/navigation flows and safer feature indexing.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

yikZero committed:
feat: enhance PrimeDashboard with custom close button and update navigation logic in PagePrimeFeatures

- Added NavCloseButton to PrimeDashboard for improved navigation control.
- Updated PagePrimeFeatures to push the PrimeDashboard directly, simplifying the navigation flow.

yikZero committed:
feat: enhance BulkCopyAddressesButton with improved modal handling

- Added timerUtils to manage modal display timing, ensuring the modal appears correctly after closing the action.
- Updated onPress function to close the action before showing the modal, preventing display issues.

yikZero committed:
fix: improve modal handling in BulkCopyAddressesButton

- Updated onPress function to close the action before displaying the modal, addressing potential display issues.
- Introduced a delay using timerUtils to ensure proper modal appearance after action closure.

yikZero committed:
fix: update button rendering logic in PagePrimeFeatures

- Modified the button rendering logic to conditionally display either the BackButton or CloseButton based on the showAllFeatures state, improving user navigation experience.

yikZero committed:
feat: enhance PrimeBenefitsList and PrimeDashboard with new features and translations

- Updated PrimeBenefitsList to include new benefits items for Notifications and History Export, enhancing user engagement.
- Modified PrimeDashboard to conditionally render the PrimeBanner based on subscription status, improving user experience.
- Added new translation keys for global export transaction history and multi-account notifications across multiple languages, ensuring better localization support.
- Introduced new features in the routing for Notifications and History Export, expanding the functionality of the Prime module.

yikZero committed:
fix: update navigation logic in PrimeBenefitsList to show success toast

- Modified the onPress function in PrimeBenefitsList to display a success toast message when the Prime subscription is active, enhancing user feedback.
- Removed unused import of EModalRoutes in PrimeDashboard, streamlining the code.

yikZero committed:
fix: update badge rendering logic in useBulkRevoke hook and correct translation key in zh_CN.json

- Modified the badge rendering logic in the useBulkRevoke hook to conditionally display the badge only for non-Prime users, improving clarity in user interface.
- Corrected the translation key in zh_CN.json from "逐个取消" to "逐一取消" for consistency in language usage.

yikZero committed:
fix: update unlock description translations for multiple languages

- Revised the translation for "unlock_description" in various language files to improve clarity and consistency in messaging regarding advanced features for crypto asset management.
- Adjustments made in bn.json, de.json, en_US.json, es.json, fr_FR.json, hi_IN.json, id.json, it_IT.json, ja_JP.json, ko_KR.json, pt_BR.json, pt.json, ru.json, th_TH.json, uk_UA.json, vi.json, zh_CN.json, zh_HK.json, and zh_TW.json.

yikZero committed:
Merge branch 'x' into prime

revan-zhang commented:
### :tada: **Snyk checks have passed. No issues have been found so far.**

:white_check_mark: **security/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/rayston92/pr-checks/7245f290-c45e-4c9d-a86c-f33c2c890310?refs=f80cde163ffed1c5d137587051f4e6b761b1ad16&source=prChecksComment)

:white_check_mark: **license/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/rayston92/pr-checks/7245f290-c45e-4c9d-a86c-f33c2c890310/license?refs=f80cde163ffed1c5d137587051f4e6b761b1ad16&source=prChecksComment)

yikZero commented on packages/kit/src/views/Prime/pages/PrimeDashboard/PrimeBenefitsList.tsx:
```diff
@@ -173,22 +173,88 @@ export function PrimeBenefitsList({
       <PrimeBenefitsItem
-        isComingSoon
-        icon="UndoOutline"
+        icon="FlashOutline"
         title={intl.formatMessage({
           id: ETranslations.global_bulk_revoke,
         })}
         subtitle={intl.formatMessage({
           id: ETranslations.global_bulk_revoke_desc,
         })}
+        onPress={() => {
+          if (isPrimeSubscriptionActive) {
+            Toast.success({
+              title: intl.formatMessage({
+                id: ETranslations.global_bulk_revoke,
+              }),
+            });
+          } else {
+            defaultLogger.prime.subscription.primeEntryClick({
+              featureName: EPrimeFeatures.BulkRevoke,
+              entryPoint: 'primePage',
+            });
+            navigation.navigate(EPrimePages.PrimeFeatures, {
+              showAllFeatures: true,
+              selectedFeature: EPrimeFeatures.BulkRevoke,
+              selectedSubscriptionPeriod,
+              serverUserInfo,
+            });
+          }
+        }}
+      />
```
I have left a message to zuozhuo, and I will help supplement the logic next week.

originalix commented on packages/kit/src/views/AccountManagerStacks/components/WalletEdit/BulkCopyAddressesButton.tsx:
```diff
@@ -33,9 +34,13 @@ export function BulkCopyAddressesButton({
-      onPress={async () => {
+      onPress={async (close) => {
+        close?.();
+        // 先关闭 Action 等 150ms 再弹出 Modal，避免弹出后再关闭的问题
```
统一用英文注释

yikZero replied:
已更新

yikZero committed:
fix: improve comments in BulkCopyAddressesButton for clarity

- Updated comments in the onPress function to provide clearer explanations regarding the modal handling and potential display issues.
- Ensured that the comments are now in English for better accessibility and understanding.
